.github/workflows/integration_tests.yml

```diff
@@ -253,3 +397,228 @@ jobs:
+                echo ":white_check_mark: Successfully ran test: $ex" >> $GITHUB_STEP_SUMMARY
+              else
+                echo ":x: Test failed (failed to execute): $ex" >> $GITHUB_STEP_SUMMARY
+                test_err_sum=$((test_err_sum+1))
+              fi
+            else
+              # Only run examples that are not target-specific (e.g., ionq, iqm)
+              if ! grep -q "set_target" "$ex"; then 
+                # Use --target command line option to run these examples with
+                python3 $ex --target nvcf 1> /dev/null
+                test_status=$?
+                if [ $test_status -eq 0 ]; then
+                  echo ":white_check_mark: Successfully ran test: $ex" >> $GITHUB_STEP_SUMMARY
+                else
+                  echo ":x: Test failed (failed to execute): $ex" >> $GITHUB_STEP_SUMMARY
+                  test_err_sum=$((test_err_sum+1))
+                fi
+              fi
+            fi
+          done
+          
+          set -e # Re-enable exit code error checking
+          if [ ! $test_err_sum -eq 0 ]; then
+            echo "::error::${test_err_sum} tests failed. See step summary for a list of failures"
+            exit 1
+          fi
+        shell: bash
+
+  wheel_integration_test:
+    name: Integration test using Python wheels
+    runs-on: ubuntu-latest
+    needs: [metadata, build_nvcf_image, deploy_nvcf_test_function]
+    environment: backend-validation
+    if: (success() || failure()) && (inputs.target == 'nvcf' || github.event_name == 'schedule' || inputs.target == 'nightly')
+
+    steps:
+      - name: Get code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ needs.metadata.outputs.cudaq_commit }}
+          fetch-depth: 1
+
+      - name: Install wheel
+        # Note: we only test Python 3.10 wheel in this workflow to do a quick sanity check for NVCF integration. 
+        run: |
+          workflow_id=${{ inputs.workflow_id }}
+          if [ -z "${workflow_id}" ]; then
+            workflow_id=$(gh run list --commit ${{ needs.metadata.outputs.cudaq_commit }} --workflow Publishing --status completed --event workflow_dispatch --limit 1 --json databaseId | jq '.[0].databaseId')
+          fi
+          echo "Using artifacts from workflow id $workflow_id"
+          gh run download $workflow_id
+          apt-get update && apt-get install -y --no-install-recommends python3.10 python3-pip
+          wheelfile=$(find . -name 'cuda_quantum*cp310*x86_64.whl')
+          python3.10 -m pip install $wheelfile
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+      - name: Test NVCF 
+        run: |
+          echo "### Submit to NVCF from Python wheels" >> $GITHUB_STEP_SUMMARY
+          export NVCF_API_KEY="${{ secrets.NVCF_API_KEY }}"
+          export NVCF_FUNCTION_ID="$NVCF_FUNCTION_ID"
+          export NVCF_FUNCTION_VERSION_ID="${{ needs.deploy_nvcf_test_function.outputs.nvcf_function_version_id }}"
+          export CUDAQ_LOG_LEVEL="info"
```

Comment:
It looks like you redirecting stdout to /dev/null, so the log level environment variable can be deleted.

Reply:
Thanks. I've removed this line.